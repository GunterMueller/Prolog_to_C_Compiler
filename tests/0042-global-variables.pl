:- global_variable(foo), global_variable(bar).

main :- 
	global_ref(foo, X), display(X), nl,
	global_set(foo, foo(abc, def)),
	garbage_collect,
	global_ref(foo, Y), display(Y), nl.
