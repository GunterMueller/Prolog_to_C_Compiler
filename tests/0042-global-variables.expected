0
foo(abc, def)
